From 88db387b58849fb4878a6cb7b8cee5389464b54a Mon Sep 17 00:00:00 2001
From: John Howard <howardjohn@google.com>
Date: Fri, 18 Feb 2022 10:28:52 -0800
Subject: Bump timeout on TestAuthZCheck (#37439)

Currently it will only try 1x since the command is pretty slow

see
https://storage.googleapis.com/istio-prow/pr-logs/pull/istio_istio/37414/integ-pilot_istio/1494713565586657280/build-log.txt
---
 tests/integration/pilot/istioctl_test.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/integration/pilot/istioctl_test.go b/tests/integration/pilot/istioctl_test.go
index a24a4dc4c7..a68ec6d752 100644
--- a/tests/integration/pilot/istioctl_test.go
+++ b/tests/integration/pilot/istioctl_test.go
@@ -530,7 +530,7 @@ func TestAuthZCheck(t *testing.T) {
 							}
 						}
 						return nil
-					}, retry.Timeout(time.Second*5))
+					}, retry.Timeout(time.Second*30))
 				})
 			}
 		})
-- 
2.35.3

